test/assert.js

```diff
@@ -1952,13 +1952,43 @@ describe('assert', function () {
 
     err(function() {
       assert.isAbove(null, 1, 'blah');
-    }, 'blah: expected null to be a number');
+    }, 'blah: expected null to be a number or a date');
 
     err(function() {
       assert.isAbove(1, null, 'blah');
     }, 'blah: the argument to above must be a number');
   });
 
+  it('above (dates)', function() {
+    var now = new Date();
+    var oneSecondAgo = new Date(now.getTime() - 1000);
+    assert.isAbove(now, oneSecondAgo, 'Now should be above 1 second ago');
+
+    err(function() {
+      assert.isAbove(oneSecondAgo, now);
+    }, 'expected ' + oneSecondAgo.toUTCString() + ' to be above ' + now.toUTCString());
+
+    err(function() {
+      assert.isAbove(now, now);
+    }, 'expected ' + now.toUTCString() + ' to be above ' + now.toUTCString());
+    
+    err(function() {
+      assert.isAbove(null, now);
+    }, 'expected null to be a number or a date');
+
+    err(function() {
+      assert.isAbove(now, null);
+    }, 'the argument to above must be a date');
+
+    err(function() {
+      assert.isAbove(now, 1);
+    }, 'the argument to above must be a date');
+
+    err(function() {
+      assert.isAbove(1, now, 'blah');
+    }, 'blah: the argument to above must be a number');
+  });
+
```

Comment:
@meeber All of them or some of them? I've seen this message omitted in some other tests.